api/v1alpha1/nutanix_node_types.go

```diff
@@ -6,6 +6,7 @@ package v1alpha1
 import (
 	"k8s.io/apimachinery/pkg/api/resource"
 	"k8s.io/utils/ptr"
+	"k8s.io/utils/ptr"
```

Comment:
```suggestion
```

Reply:
Thanks. I also need to re-generate code.